docs/ros/tutorials/simulator/install.mdx

```diff
@@ -47,7 +47,7 @@ mkdir ~/clearpath_ws/src -p
 **2. Import dependencies**
 
 ```
-source /opt/ros/humble/setup.bash
+source /opt/ros/jazzy/setup.bash
 sudo apt install python3-vcstool
 cd ~/clearpath_ws
 wget https://raw.githubusercontent.com/clearpathrobotics/clearpath_simulator/main/dependencies.repos
```

Comment:
Should be: `https://raw.githubusercontent.com/clearpathrobotics/clearpath_simulator/jazzy/dependencies.repos`

Reply:
Updated